x-pack/plugins/embeddable_enhanced/public/embeddables/dynamic_action_storage.ts

```diff
@@ -115,8 +104,8 @@ export class EmbeddableActionStorage extends AbstractActionStorage {
   }
 
   private getEventsFromEmbeddable() {
```

Comment:
nit, how about renaming `getEventsFromEmbeddable` to just `getEvents`?

Reply:
Ahh, good call! Done in 5710d14c8493e2456d4b639ba1e18ee1012b256f